README.md

```diff
@@ -25,86 +25,13 @@
 
-For more information about Gossamer and the Polkadot Host, check out [Gossamer Wiki](https://github.com/ChainSafe/gossamer/wiki).
-
-## Package Architecture
-
-Gossamer includes [node implementations](#node-implementations) for major networks within the Polkadot ecosystem, [node services](#node-services) that can be used to build and run node implementations, and a collection of [modular packages](#modular-packages) that can be used to build and run node services and other supporting tools.
-
-### Node Implementations
-
-Gossamer includes node implementations in development for Gossamer Testnet and Kusama Network.
-
-| package           | description |
-|-                  |-            |
-| `node/gssmr`      | a full node implementation and rpc server for Gossamer Testnet |
-| `node/ksmcc`      | a full node implementation and rpc server for Kusama Network |
-
-### Node Services
-
-Gossamer includes node services used to build and run node implementations with a shared base protocol (currently each node implementation uses a set of shared node services that make up the base implementation for the Polkadot Host).
-
-| package           | description |
-|-                  |-            |
-| `dot/core`        | orchestrate service interactions |
-| `dot/network`     | peer-to-peer service using libp2p |
-| `dot/rpc`         | optional service for RPC server |
-| `dot/state`       | storage service for chain state |
-
-### Modular Packages
-
-Gossamer includes a collection of modular packages used to build and run node services and other supporting tools.
-
-| package           | description |
-|-                  |-            |
-| `lib/babe`        | BABE implementation |
-| `lib/blocktree`   | blocktree implementation |
-| `lib/common`      | common types and functions |
-| `lib/crypto`      | crypto keypair implementations |
-| `lib/database`    | generic database using badgerDB |
-| `lib/grandpa`     | GRANDPA implementation |
-| `lib/keystore`    | library for managing keystore |
-| `lib/runtime`     | WASM runtime integration using Wasmer |
-| `lib/scale`       | SCALE encoding and decoding |
-| `lib/services`    | common interface for node services |
-| `lib/transaction` | library for transaction queue |
-| `lib/trie`        | modified merkle-patricia trie implementation |
-
-## Running Gossamer
-
-The `gossamer` command can be used to run node implementations that are within this repository.
-
-### Prerequisites
-
-go 1.13.7
-
-### Installation
-
-```
-go get -u github.com/ChainSafe/gossamer
-```
-
-### Build Gossamer
-
-```
-make gossamer
-```
-
-### Run Gossamer Node
-
-```
-./bin/gossamer init
-```
-
-```
-./bin/gossamer
-```
```

Comment:
I think we should leave quick instructions like this in the readme

Reply:
agreed, I think I meant to just snip out the architecture stuff

Reply:
done :+1: